test/playback.test.js

```diff
@@ -167,8 +167,6 @@ QUnit.test('Live DASH', function(assert) {
 });
 
 QUnit.test('loops', function(assert) {
-  assert.timeout(5000);
```

Comment:
this should be kept in since it stops the tests from getting stuck

Reply:
Ah yes, I removed this way earlier because I was seeing something weird (that went away).  I'll re-add this.